service/src/main/resources/config/verily/sdd_refresh0323/entity/person.json

```diff
@@ -14,6 +14,7 @@
     { "type": "SIMPLE", "name": "biovu_sample_is_nonshippable", "dataType": "BOOLEAN" },
     { "type": "SIMPLE", "name": "biovu_sample_has_plasma", "dataType": "BOOLEAN" }
   ],
+  "frequentFilterAttributes": [ "gender", "race", "age" ],
```

Comment:
Nothing to do here now but we should clean this up to avoid duplicating what's in the UI config for breakdowns.

Reply:
Agreed.